Source/Diagnostics/ComputeDiagFunctors/CellCenterFunctor.H

```diff
@@ -13,11 +13,14 @@ public:
     /** Constructor.
      * \param[in] mf_src source multifab.
      * \param[in] lev level of multifab. Used for averaging in rz.
+     * \param[in] crse_ratio for interpolating field values from src_mf
+                  to the coarsened dst_mf
```

Comment:
```suggestion
     * \param[in] crse_ratio for interpolating field values from simulation MultiFab mf_src
                  to diags MultiGab mf_dst
```

Reply:
Could you address this comment?

Reply:
This has a spelling mistake for MultiGab instead of MultiFab. Was going to correct it in the file directly.